po/zh-Hant.po

```diff
@@ -7257,14 +7258,16 @@ msgstr ""
 
 #: src/generics.md:1
 msgid "# Generics"
-msgstr ""
+msgstr "# 泛型 (Generics)"
```

Comment:
Not sure if there's a style guide about whether to include the English name in titles.

Reply:
Updated.